lib/parameters.py

```diff
@@ -58,6 +58,42 @@ def gen_avc_cqp_parameters(spec, profiles):
   params = gen_avc_cqp_variants(spec, profiles)
   return keys, params
 
+def gen_avc_cqp_seek_variants(spec, profiles):
+  for case, params in spec.items():
+    variants = copy.deepcopy(params.get("variants", dict()).get("cqp", None))
+    if variants is None:
+      keys = ["gop", "slices", "bframes", "qp", "quality", "profile"]
+      product  = list(itertools.product([1], [1], [0], [14, 28], [1, 4, 7], profiles))  # I, single-slice
+      product += list(itertools.product([30], [1], [0], [14, 28], [1, 4, 7], profiles)) # IP, single-slice
+      product += list(itertools.product([30], [4], [2], [14, 28], [1, 4, 7], profiles)) # IPB, multi-slice
+      variants = [dict(zip(keys, vals)) for vals in product]
```

Comment:
We don't need defaults generated if the variant is not defined in test config.

Also, we should not reuse the "cqp" variant key.  It is meant for the "cqp" tests and has a different parameter signature.  If we reuse it, then there will suddenly be double amount of cases generated from current test configs (a set for cqp.test and a set for cqp_seek.test) that do the exact same thing.  

Same concern for "vbr" and "tcbr" added here too.